packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -38,140 +13,3 @@ export const getSplitButtonVerticalDividerClassNames = memoizeFunction(
     });
   }
 );
-
-export const getContextualMenuClassNames = memoizeFunction(
```

Comment:
These older `getStyles` type calls have been refactored to their corresponding `*.styles.ts` file.